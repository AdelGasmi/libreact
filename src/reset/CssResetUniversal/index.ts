import createCssResetComponent from '../createCssResetComponent';

export const css = {
  '*': {
    'vertical-align': 'baseline',
    fw: 'inherit',
    ff: 'inherit',
    fs: 'inherit',
    fz: '100%',
    bd: '0 none',
    out: 0,
    pad: 0,
    mar: 0,
  },
};

const CssResetUniversal = createCssResetComponent(css);

export default CssResetUniversal;
