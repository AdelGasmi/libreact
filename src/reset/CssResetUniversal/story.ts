import {createElement as h} from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';
import {linkTo} from '@storybook/addon-links';
import CssResetUniversal from '.';

storiesOf('reset', module)
  .add('CssResetUniversal', () =>
    h(CssResetUniversal)
  );
