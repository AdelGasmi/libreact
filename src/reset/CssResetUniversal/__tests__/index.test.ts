import {createElement as h} from 'react';
import {renderToStaticMarkup} from 'react-dom/server';
import CssResetUniversal from '..';

describe('<CssResetUniversal>', () => {
  it('renders as expected', () => {
    const html = renderToStaticMarkup(h(CssResetUniversal));

    expect(html).toMatchSnapshot();
  });
});
