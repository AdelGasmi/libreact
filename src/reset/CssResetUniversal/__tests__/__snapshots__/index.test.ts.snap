// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssResetUniversal> renders as expected 1`] = `"<style>*{vertical-align:baseline;font-weight:inherit;font-family:inherit;font-style:inherit;font-size:100%;border:0 none;outline:0;padding:0;margin:0;}</style>"`;
